tests/api/test_job.py

```diff
@@ -32,14 +34,31 @@ def test_incomplete_job_raises_on_result_access(self, connection):
             AttributeError,
             match="The result is undefined for jobs that are not completed",
         ):
-            _ = job.result
+            job.result
```

Comment:
@pauljxtan I see, was the linter complaining if you didn't assign the property to something? In that case, I might defer to the linter (sorry for making pointless work :laughing:)

Reply:
Haha yes it was :slightly_smiling_face: